spec/examples/gulpfile.js

```diff
@@ -1,22 +1,23 @@
 /**
  * an example gulpfile to make ant-less existdb package builds a reality
  */
-const fs = require('fs')
 const { src, dest, watch, series, parallel, lastRun } = require('gulp')
-const { createClient } = require('@existdb/gulp-exist')
-const { connect } = require('@existdb/node-exist')
+const { createClient } = require('../../index')
```

Comment:
Should this not be `require(‘@existdb/gulp-exist’)` in the example file, as it’s what users are likely to do?

Reply:
Yeah, you are probably right. I changed it to be able to see the code hints.